# YLB @ Axios provided this patch in openembedded-hawk
# There's not a cleaner way to get at this variable post-patch
# in bitbake so far as BTG can tell, either.
diff --git a/librtlsdr.pc.in b/librtlsdr.pc.in
index 5e55049..6c42f8c 100644
--- a/librtlsdr.pc.in
+++ b/librtlsdr.pc.in
@@ -6,6 +6,7 @@ includedir=@includedir@
 Name: RTL-SDR Library
 Description: C Utility Library
 Version: @VERSION@
-Cflags: -I${includedir}/ @RTLSDR_PC_CFLAGS@
+# YLB - ugly hack should fix later.
+Cflags: -I${includedir}/ -I${includedir}/libusb-1.0
 Libs: -L${libdir} -lrtlsdr -lusb-1.0
 Libs.private: @RTLSDR_PC_LIBS@
